luzer/custom_mutator_lib.c

```diff
@@ -12,13 +12,8 @@
 
 #include "luzer.h"
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-int luaL_error(lua_State *L, const char *fmt, ...);
-size_t lua_objlen(lua_State *L, int index);
-#ifdef __cplusplus
-} /* extern "C" */
+#if LUA_VERSION_NUM == 501
+#define lua_rawlen(L, i) lua_objlen(L, i)
```

Comment:
Minor: It's better to use it like `lua_objlen((L), (i))` to avoid surprises from the macro.